interfaces/builtin/desktop_calendar_service.go

```diff
@@ -0,0 +1,138 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const desktopCalendarServiceSummary = `allows communication with Evolution Data Service Calendar`
+
+const desktopCalendarServiceBaseDeclarationSlots = `
+  desktop-calendar-service:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
```

Comment:
Since this is an implicit classic interface we don't need a deny-connection constrain (we trust the classic distro's slot implementation), so yes, deny-auto-connection is sufficient.